locations.xml

```diff
@@ -318999,6 +319002,7 @@
       <LocationId>coruna</LocationId>
     </FallbackLocations>
     <Names>
+      <Name language="Arabic" value="Qarǧiṭaẗ" comment="found in 4118Htm" />
```

Comment:
This location (`coruna_county`) already inherits names from location `coruna`, so no need to define it in both places. If it's used for Coruña generically, put it under `coruna`, if it's specific to just the county/region, put it under `coruna_county`.